### PR TITLE
Add -o option to overwrite target files

### PR DESCRIPTION
**Pull request by ningyougang**
If target directory already has unzip files, the unzip command will
give tip:[y]es, [n]o, [A]ll, [N]one, [r]ename, so it is better to
use the -o option to overwrite files.

## Related issue and scope

~~- [ ] I opened an issue to propose and discuss this change (#????)~~

## My changes affect the following components

- [ ] API
- [ ] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [ ] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [ ] Tests
- [ ] Deployment
- [x] CLI
- [ ] General tooling
- [ ] Documentation

## Types of changes

- [x] Bug fix (generally a non-breaking change which closes an issue).
- [ ] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:

- [x] I signed an [Apache CLA](https://github.com/apache/incubator-openwhisk/blob/master/CONTRIBUTING.md).
- [x] I reviewed the [style guides](https://github.com/apache/incubator-openwhisk/wiki/Contributing:-Git-guidelines#code-readiness) and followed the recommendations (Travis CI will check :).
~~- [ ] I added tests to cover my changes.~~
~~- [ ] My changes require further changes to the documentation.~~
~~- [ ] I updated the documentation where necessary.~~

### CODE REVIEW

**Comment by ningyougang** [comment]
In my local build env. the second build will be failed due to already exist the files, the error like below
```
TASK [cli : Unarchive the individual zipfiles into binaries] *******************
task path: /home1/irteamsu/.jenkins/workspace/lambda-pr-build/ansible/roles/cli/tasks/deploy.yml:71
Thursday 15 March 2018  15:26:20 +0900 (0:00:00.252)       0:08:36.533 ******** 
Using module file /usr/lib/python2.7/site-packages/ansible/modules/commands/command.py
....
<172.17.0.1> EXEC /bin/sh -c 'echo ~ && sleep 0'
<172.17.0.1> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home1/irteamsu/.ansible/tmp/ansible-tmp-1521095180.8-192726596548796 `" && echo ansible-tmp-1521095180.8-192726596548796="` echo /home1/irteamsu/.ansible/tmp/ansible-tmp-1521095180.8-192726596548796 `" ) && sleep 0'
<172.17.0.1> PUT /tmp/tmp9s_M9e TO /home1/irteamsu/.ansible/tmp/ansible-tmp-1521095180.8-192726596548796/command.py
<172.17.0.1> EXEC /bin/sh -c 'chmod u+x /home1/irteamsu/.ansible/tmp/ansible-tmp-1521095180.8-192726596548796/ /home1/irteamsu/.ansible/tmp/ansible-tmp-1521095180.8-192726596548796/command.py && sleep 0'
<172.17.0.1> EXEC /bin/sh -c '/usr/bin/python /home1/irteamsu/.ansible/tmp/ansible-tmp-1521095180.8-192726596548796/command.py; rm -rf "/home1/irteamsu/.ansible/tmp/ansible-tmp-1521095180.8-192726596548796/" > /dev/null 2>&1 && sleep 0'
failed: [172.17.0.1] (item={u'uid': 11000, u'woth': False, u'mtime': 1521095177.3356953, u'inode': 72173708, u'isgid': False, u'size': 3158024, u'isuid': False, u'isreg': True, u'gid': 11000, u'ischr': False, u'wusr': True, u'xoth': True, u'islnk': False, u'nlink': 1, u'issock': False, u'rgrp': True, u'path': u'/tmp/wskconf/nginx/cli/go/download/windows/386/OpenWhisk_CLI-windows-32bit.zip', u'xusr': True, u'atime': 1521002363.409153, u'isdir': False, u'ctime': 1521095177.3356953, u'isblk': False, u'wgrp': False, u'xgrp': True, u'dev': 2050, u'roth': True, u'isfifo': False, u'mode': u'0755', u'rusr': True}) => {
    "changed": true, 
    "cmd": "unzip /tmp/wskconf/nginx/cli/go/download/windows/386/OpenWhisk_CLI-windows-32bit.zip -d /tmp/wskconf/nginx/cli/go/download/windows/386", 
    "delta": "0:00:00.014804", 
    "end": "2018-03-15 15:26:20.971211", 
    "invocation": {
        "module_args": {
            "_raw_params": "unzip /tmp/wskconf/nginx/cli/go/download/windows/386/OpenWhisk_CLI-windows-32bit.zip -d /tmp/wskconf/nginx/cli/go/download/windows/386", 
            "_uses_shell": true, 
            "chdir": null, 
            "creates": null, 
            "executable": null, 
            "removes": null, 
            "stdin": null, 
            "warn": true
        }
    }, 
    "item": {
        "atime": 1521002363.409153, 
        "ctime": 1521095177.3356953, 
        "dev": 2050, 
        "gid": 11000, 
        "inode": 72173708, 
        "isblk": false, 
        "ischr": false, 
        "isdir": false, 
        "isfifo": false, 
        "isgid": false, 
        "islnk": false, 
        "isreg": true, 
        "issock": false, 
        "isuid": false, 
        "mode": "0755", 
        "mtime": 1521095177.3356953, 
        "nlink": 1, 
        "path": "/tmp/wskconf/nginx/cli/go/download/windows/386/OpenWhisk_CLI-windows-32bit.zip", 
        "rgrp": true, 
        "roth": true, 
        "rusr": true, 
        "size": 3158024, 
        "uid": 11000, 
        "wgrp": false, 
        "woth": false, 
        "wusr": true, 
        "xgrp": true, 
        "xoth": true, 
        "xusr": true
    }, 
    "msg": "non-zero return code", 
    "rc": 1, 
    "start": "2018-03-15 15:26:20.956407", 
    "stderr": "replace /tmp/wskconf/nginx/cli/go/download/windows/386/wsk.exe? [y]es, [n]o, [A]ll, [N]one, [r]ename:  NULL\n(EOF or read error, treating as \"[N]one\" ...)", 
    "stderr_lines": [
        "replace /tmp/wskconf/nginx/cli/go/download/windows/386/wsk.exe? [y]es, [n]o, [A]ll, [N]one, [r]ename:  NULL", 
        "(EOF or read error, treating as \"[N]one\" ...)"
    ], 
    "stdout": "Archive:  /tmp/wskconf/nginx/cli/go/download/windows/386/OpenWhisk_CLI-windows-32bit.zip", 
    "stdout_lines": [
        "Archive:  /tmp/wskconf/nginx/cli/go/download/windows/386/OpenWhisk_CLI-windows-32bit.zip"
    ]
}
...
```

**Comment by rabbah** [comment]
@jonpspri can you review this please?

File: ansible/roles/cli/tasks/deploy.yml [hunk 1]
```diff
@@ -69,5 +69,5 @@
 # Use old good unzip instead of unarchive due to a known issue on MacOS
 # https://github.com/ansible/ansible-modules-core/issues/3952
 - name: "Unarchive the individual zipfiles into binaries"
-  shell: unzip {{ item.path }} -d {{ item.path | dirname }}
+  shell: unzip -o {{ item.path }} -d {{ item.path | dirname }}
```
Comment (jonpspri) [inline ansible/roles/cli/tasks/deploy.yml]: LGTM.  I ran into this same problem yesterday, but didn't have time to research the unzip option.

File: ansible/roles/kafka/tasks/deploy.yml [hunk 2]
```diff
@@ -79,7 +79,7 @@
     env: "{{ kafka_env_vars }}"
     ports:
       - "{{ kafka.port + groups['kafkas'].index(inventory_hostname) }}:{{ kafka.port + groups['kafkas'].index(inventory_hostname) }}"
-      - "{{ kafka.advertisedPort  + groups['kafkas'].index(inventory_hostname) }}:{{ kafka.advertisedPort + groups['kafkas'].index(inventory_hostname) }}"
+      - "{{ kafka.advertisedPort + groups['kafkas'].index(inventory_hostname) }}:{{ kafka.advertisedPort + groups['kafkas'].index(inventory_hostname) }}"
```
Comment (jonpspri) [inline ansible/roles/kafka/tasks/deploy.yml]: Would normally ask for a separate 'typo' PR for this, but seems harmless enough to include it with other ansible changes.

**Pull request merged**